tests/backup/backup_restore_basic_test.go

```diff
@@ -1318,3 +1318,128 @@ var _ = Describe("{BackupSyncBasicTest}", func() {
+		})
+
+		Step("Creating cloud credentials and registering backup location", func() {
+			log.InfoD("Creating cloud credentials and registering backup location")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to px-central-admin ctx")
+			for _, provider := range providers {
+				cloudCredUID = uuid.New()
+				cloudCredUidList = append(cloudCredUidList, cloudCredUID)
+				backupLocationUID = uuid.New()
+				credName = fmt.Sprintf("autogenerated-cred-%v", time.Now().Unix())
+				err = CreateCloudCredential(provider, credName, cloudCredUID, orgID, ctx)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating cloud credential named %s", credName))
+				log.InfoD("Created Cloud Credentials with name - %s", credName)
+				backupLocationName = fmt.Sprintf("autogenerated-backup-location-%v", time.Now().Unix())
+				backupLocationMap[backupLocationUID] = backupLocationName
+				err := CreateBackupLocation(provider, backupLocationName, backupLocationUID, credName, cloudCredUID, getGlobalBucketName(provider), orgID, "")
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating backup location %s", backupLocationName))
+			}
+		})
+		Step("Configure source and destination clusters with px-central-admin ctx", func() {
+			log.InfoD("Configuring source and destination clusters with px-central-admin ctx")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			err = CreateSourceAndDestClusters(orgID, "", "", ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of source [%s] and destination [%s] clusters with px-central-admin ctx", SourceClusterName, destinationClusterName))
+			appClusterName := destinationClusterName
+			clusterStatus, err := Inst().Backup.GetClusterStatus(orgID, appClusterName, ctx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", appClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", appClusterName))
+			clusterUid, err = Inst().Backup.GetClusterUID(ctx, orgID, appClusterName)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching [%s] cluster uid", appClusterName))
+			log.InfoD("Uid of [%s] cluster is %s", appClusterName, clusterUid)
+		})
+		Step("Taking a backup of application", func() {
+			log.InfoD("Taking a backup of application")
+			ctx, err := backup.GetAdminCtxFromSecret()
+			log.FailOnError(err, "Unable to fetch px-central-admin ctx")
+			namespaceLabel := MapToKeyValueString(labelMap)
+			backupName = fmt.Sprintf("%s-%v", "backup", time.Now().Unix())
+			err = CreateBackupWithNamespaceLabel(backupName, SourceClusterName, backupLocationName, backupLocationUID,
+				nil, orgID, clusterUid, "", "", "", "", namespaceLabel, ctx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying backup [%s] creation with label [%s]", backupName, namespaceLabel))
```

Comment:
I can see only few labels from this output. Is this latest one ?
https://jenkins.pwx.dev.purestorage.com/job/Users/job/Sumit/job/Custom-Pipelines/job/px-backup-on-demand-system-test-byoc/765/consoleFull

```
20:26:48  INFO[2023-03-29 14:56:47] Verifying : Description : Verifying backup [backup-1680101776] creation with label [3-8af10fa7-5427-4a9e-b708-8fc53f3440b3=c38c2986-c3d6-4a42-85e7-6705ebde6161,4-5fe729ea-15dd-4e98-9841-76cf0c4258c8=2101e216-f072-42f9-aab7-2de4091bf4c5,8-33a63c91-6067-41c2-a1f7-bcdb5155fbec=11303821-8675-4605-86fb-22d0d2a2a9e9,9-887fcc2e-7b39-4bef-8256-ed8d846a6c40=6adec6e2-1213-48c4-aa08-82505183c2bc,0-2c2abbd4-ab34-4e14-864c-898f5c426904=390a566d-7598-491c-8f30-12b4c990dda6,1-36df360c-08b3-4544-ad2e-653a2c80de48=876a942a-d366-43bf-963b-741212ba4fe0,2-ff9ee144-8569-4fd4-97d7-d657defbe2d3=1d646a15-65b2-4232-80e7-693a8615f458,5-1a7568da-b760-4fa5-bcaa-d787318704f5=ce14d502-5a53-42a3-aad4-3e20d46fb5a8,6-f4da76a7-2c7a-44ed-b261-b6de15d4775f=19f18e44-4cfe-4fcf-9bba-672081101e96,7-7075e507-2431-41db-bdc7-1d67f637cdec=e5b5d1dc-4660-4ba2-b49d-48b9df0b07c7] 
```